src/hermes/utils.py

```diff
@@ -0,0 +1,16 @@
+# SPDX-FileCopyrightText: 2023 Helmholtz-Zentrum Dresden-Rossendorf (HZDR)
+#
+# SPDX-License-Identifier: Apache-2.0
+
+# SPDX-FileContributor: David Pape
+
+from importlib.metadata import metadata
+
+
+hermes_metadata = metadata("hermes").json
```

Comment:
**question:** Why this extra conversion step here instead of using the PEP 314 directly?

Reply:
No reason. I changed it in 980270a27a6be1cef6bd86076281fc1d6e41e022